java/src/main/java/com/plaid/quickstart/resources/AccessTokenResource.java

```diff
@@ -0,0 +1,64 @@
+package com.plaid.quickstart.resources;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.plaid.client.PlaidClient;
+import com.plaid.client.request.ItemPublicTokenExchangeRequest;
+import com.plaid.client.response.ItemPublicTokenExchangeResponse;
+import com.plaid.quickstart.QuickstartApplication;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.FormParam;
+import javax.ws.rs.core.MediaType;
+
+import retrofit2.Response;
+
+@Path("/")
+public class AccessTokenResource {
+    private PlaidClient plaidClient;
+
+    public AccessTokenResource(PlaidClient _plaidClient) {
+        plaidClient = _plaidClient;
+    }
+
+    @POST
+    @Path("get_access_token")
+    @Produces("application/json")
+    public Object getAccessToken(@FormParam("public_token") String publicToken) throws IOException {
+        Response<ItemPublicTokenExchangeResponse> itemResponse = plaidClient.service()
+            .itemPublicTokenExchange(new ItemPublicTokenExchangeRequest(publicToken))
+            .execute();
+
+        QuickstartApplication.accessToken = itemResponse.body().getAccessToken();
```

Comment:
im kind of confused why are we setting a static variable every time

Reply:
how should we pass accessToken to other resources?